wikipron/config.py

```diff
@@ -119,13 +165,16 @@ def _get_casefold(self, casefold: bool) -> Callable[[str], str]:
         return str.casefold if casefold else lambda word: word  # noqa: E731
 
     def _get_process_pron(
-        self, no_stress: bool, no_syllable_boundaries: bool
+        self, no_stress: bool, no_syllable_boundaries: bool, no_segment: bool
     ) -> Callable[[str], str]:
+
         processors = []
         if no_stress:
             processors.append(functools.partial(re.sub, r"[ˈˌ]", ""))
         if no_syllable_boundaries:
             processors.append(functools.partial(re.sub, r"\.", ""))
+        if not no_segment:
+            processors.append(functools.partial(_parse_combining_modifiers))
```

Comment:
The test failure is due to this line. Each "processor" is a one-argument function, and `_parse_combining_modifiers` is already one. So we don't need the `functools.partial` bit.